src/commands/command.ts

```diff
@@ -39,11 +38,9 @@ export default abstract class Command {
     if (await message.user.hasRole(message.channel, this.role)) {
       await this.action(message, match);
       const time = Date.now() - message.timestamp.valueOf();
-      if (!(this instanceof CommandGroup)) {
-        bot.logger.debug(
-          `Command '${this.name}' executed on channel ${message.channel.label} in ${time} ms.`,
-        );
-      }
+      bot.logger.debug(
+        `Command '${this.name}' executed on channel ${message.channel.label} in ${time} ms.`,
+      );
```

Comment:
@TimJentzsch I don't understand why this check existed but it was the root of a circular dependency. If it's necessary we will need a different type of check

Reply:
I think this was to avoid duplicate log messages for command groups